apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -167,6 +167,110 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
     state
   end
 
+  deffixture invalid_piggyback_on_input(
+               alice,
+               processor_filled,
+               transactions,
+               ife_tx_hashes,
+               competing_transactions
+             ) do
+    tx = hd(transactions)
+    comp1 = hd(competing_transactions)
```

Comment:
Pls mind source code is mostly read - try to avoid shortcuts
```suggestion
    competitor = hd(competing_transactions)
```

Please check other tests :point_down: